release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1584,6 +1584,12 @@ metadata:
 [id="ocp-4-17-rhcos-bug-fixes_{context}"]
 ==== {op-system-first}
 
+* Previously, LUKS encryption on a system using 512 emulation disks caused provisioning to fail at the `ignition-ostree-growfs` step because of an `sfdisk` alignment issue. With this release, the `ignition-ostree-growfs` script detects this situation and fixes the alignment automatically. As a result, the system no longer fails during provisioning. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
+
+* Previously, a bug in the `growpart` utility caused a LUKS device to lock. This prevented the system from booting and entering into an emergency mode. With this release, the call to the `growpart` utility is removed and the system successfully boots without issue. (link:https://issues.redhat.com/browse/OCPBUGS-33124[*OCPBUGS-33124*])
```

Comment:
Per the bug, I think this is more accurate:
```suggestion
* Previously, a bug in the `growpart` utility caused a LUKS device to lock. This caused the system to boot into an emergency mode. With this release, the call to the `growpart` utility is removed and the system successfully boots without issue. (link:https://issues.redhat.com/browse/OCPBUGS-33124[*OCPBUGS-33124*])
```

Reply:
Good catch - ty!